Makefile

```diff
@@ -58,6 +58,8 @@ IMG_VERSION ?= $(VERSION)-$(TAG)
 BASE_IMG       := $(REGISTRY)/$(REPOSITORY)/$(IMG_NAME)
 OPERATOR_IMAGE ?= $(BASE_IMG):$(IMG_VERSION)
 
+show-operator-image:
```

Comment:
Nit: `print-operator-image-name`  might be more descriptive of what this target does.

Reply:
I tried to be consistent with the `show-image` target of the `.ci/Makefile` but I prefer `print-`. Let's rename the two targets. 
I feel that the `name` suffix is a little noisy. We could also argue that the full image name is composed by a registry name, a repository name and an image name, so it's always a little ambiguous. 
I'm going to keep `print-ci-image` and `print-operator-image`.